lib/Parse/ParsePattern.cpp

```diff
@@ -431,6 +431,13 @@ mapParsedParameters(Parser &parser,
                            param.FirstName, param.FirstNameLoc);
     }
 
+    // Warn when an unlabeled parameter follows a variadic parameter
+    if (ellipsisLoc.isValid() && elements.back()->isVariadic() &&
+        param.FirstName.empty()) {
```

Comment:
@rintaro,

 `ellipsisLoc.isValid()` is required to prevent crashing when `elements` is empty.